compiler/il/AliasSetInterface.hpp

```diff
@@ -36,24 +36,16 @@
 
 
 template <class AliasSetInterface>
-class TR_AliasSetInterface : public AliasSetInterface {
+class TR_AliasSetInterface {
 public:
 
-  TR_AliasSetInterface(TR::SymbolReference *symRef, bool isDirectCall = false,
-                       bool includeGCSafePoint = false)
-    : AliasSetInterface(symRef),
-      _isDirectCall(isDirectCall),
-      _includeGCSafePoint(includeGCSafePoint) {
-  }
-
-  TR_AliasSetInterface(TR::Node *node, bool isDirectCall = false, bool includeGCSafePoint = false) :
-    AliasSetInterface(node),
+  TR_AliasSetInterface(bool isDirectCall = false, bool includeGCSafePoint = false) :
```

Comment:
Not quite. It's because of the inversion of the class hierarchy. 

`TR_AliasSetInterface` used to be in the middle of the hierarchy and needed to pass the node to it's parent. Now that it's at the top, it doesn't need to know about the node (or symref for that matter). It is the derived class `TR_NodeAliasSetInterface` that now keeps track of the node.